main.js

```diff
@@ -0,0 +1,120 @@
+const API_KEY = "6d29bbab";
+
+document.getElementById("form-submit").addEventListener("submit", getIp);
+let page = 1;
+let searchWord = document.getElementById("searchWord");
+function getIp(e) {
```

Comment:
I have the same question 

https://github.com/k1ddd/hillel-front-end-pro-/pull/14#discussion_r1028971831